libcoz/CMakeLists.txt

```diff
@@ -37,6 +35,6 @@ if(INSTALL_COZ)
     install(TARGETS coz
         ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
         LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
-        RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
-        PUBLIC_HEADER DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/coz)
+        RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
+    install(DIRECTORY ${PROJECT_SOURCE_DIR}/include/. TYPE INCLUDE)
```

Comment:
The `TYPE` argument to `install(DIRECTORY)` was introduced in CMake 3.14. Yet the top-level CMakeLists.txt requires only CMake 3.4. It is probably best to use `DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}` instead as to avoid having to increase the required CMake version beyond what is currently available on older Debian/Ubuntu systems.

Reply:
Done, thanks!
https://github.com/plasma-umass/coz/pull/178/commits/fafd230bbbc8b4ee8ecbd31efb3a1c71ea5dee11

Reply:
@Corristo Thank you for comprehensive code-review, I have learning a few new things!